crates/oxc_linter/src/rules/typescript/no_empty_interface.rs

```diff
@@ -66,25 +79,134 @@ fn test() {
     use crate::tester::Tester;
 
     let pass = vec![
-        "interface Foo { name: string; }",
-        "interface Foo { name: string; }
-        interface Bar { age: number; }
-        // valid because extending multiple interfaces can be used instead of a union type
-        interface Baz extends Foo, Bar {}",
+        (
+            "
+			interface Foo {
+			  name: string;
+			}
+			    ",
+            None,
+        ),
+        (
+            "
+			interface Foo {
+			  name: string;
+			}
+
+			interface Bar {
+			  age: number;
+			}
+
+			// valid because extending multiple interfaces can be used instead of a union type
+			interface Baz extends Foo, Bar {}
+			    ",
+            None,
+        ),
+        (
+            "
+			interface Foo {
+			  name: string;
+			}
+
+			interface Bar extends Foo {}
+			      ",
+            Some(serde_json::json!([{ "allow_single_extends": true }])),
+        ),
+        (
+            "
+			interface Foo {
+			  props: string;
+			}
+
+			interface Bar extends Foo {}
+
+			class Bar {}
+			      ",
+            Some(serde_json::json!([{ "allow_single_extends": true }])),
+        ),
     ];
 
     let fail = vec![
-        "interface Foo {}",
-        "interface Foo { props: string; } interface Bar extends Foo {} class Baz {}",
-        "interface Foo { props: string; } interface Bar extends Foo {} class Bar {}",
-        "interface Foo { props: string; } interface Bar extends Foo {} const bar = class Bar {};",
-        "interface Foo { name: string; } interface Bar extends Foo {}",
-        "interface Foo extends Array<number> {}",
-        "interface Foo extends Array<number | {}> {}",
-        "interface Bar { bar: string; } interface Foo extends Array<Bar> {}",
-        "type R = Record<string, unknown>; interface Foo extends R {}",
-        "interface Foo<T> extends Bar<T> {}",
-        "declare module FooBar { type Baz = typeof baz; export interface Bar extends Baz {} }",
+        ("interface Foo {}", None),
```

Comment:
Looks also need to add [`interface Foo extends {}`](https://github.com/typescript-eslint/typescript-eslint/blob/a6ab2cb9be5a09498e953896e7fad04b2c53ea42/packages/eslint-plugin/tests/rules/no-empty-interface.test.ts#L64)

Reply:
mmm, this was not added by codegen